js/apps/system/_admin/aardvark/APP/react/src/views/collections/addCollection/AddCollectionModal.tsx

```diff
@@ -25,7 +25,7 @@ const INITIAL_VALUES = {
   distributeShardsLike: "",
   isSatellite: false,
   smartJoinAttribute: ""
-} as Partial<CreateCollectionOptions> & Record<string, any>;
```

Comment:
`staisfies` operator throws typescript errors on build for me. Please revert this